lib/cocoapods-packager/pod_utils.rb

```diff
@@ -240,6 +240,7 @@ def write_pod_project(dynamic_project, dynamic_sandbox)
 
           # Edit search paths so that we can find our dependency headers
           dynamic_project.targets.first.build_configuration_list.build_configurations.each do |config|
+            config.build_settings['FRAMEWORK_SEARCH_PATHS'] = "$(inherited) #{Dir.pwd}/Pods/Static/**"
```

Comment:
I have a pod with a dynamic framework in its dependencies. And this glob works fine. Because the packager place dependencies into `Pods/Static` this glob could not be more concrete.